idaes/models_extra/gas_solid_contactors/properties/oxygen_iron_OC_oxidation/solid_phase_thermo.py

```diff
@@ -506,21 +585,31 @@ def density_particle_constraint(b):
 
     def _cp_mol_comp(self):
         # Pure component solid heat capacities
+        units_meta = self._params.get_metadata().derived_units
+        units_cp_mol = (
```

Comment:
As before, you can get this directly from `unit_meta`.

See line 249 in https://github.com/IDAES/idaes-pse/blob/main/idaes/core/base/property_meta.py for reference about which units are already defined (and if something is missing, you should add it there).

Reply:
Thanks for looking at this and the other units tags, I'll cross-reference `property_meta.py` and reduce to derived units where I can.